apps/api-extractor/src/generators/ApiModelGenerator.ts

```diff
@@ -709,6 +712,7 @@ export class ApiModelGenerator {
       }
       const isOptional: boolean =
         (astDeclaration.astSymbol.followedSymbol.flags & ts.SymbolFlags.Optional) !== 0;
+      const isReadonly: boolean = this._determineReadonly(astDeclaration);
```

Comment:
Method declarations cannot be `readonly`, remove? https://www.typescriptlang.org/play?#code/MYGwhgzhAECiAeYC2AHEBTaBvAUNf0ATumACYD2AdiAJ7QBm55AFAJQBc0AbuQJanYAvjmE5elAC7pC9MMEwJkadACZseAsTJVaDJm049+AbhFA. And update the mixin usage accordingly.